### DISCUSSION TITLE
HM-MOD-RPI-PCB wird von RPI2B V1.1 nicht erkannt

### DISCUSSION THREAD

**Discussion by DiaAlba** [OpenCCU/OpenCCU#1936] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
ich habe mich seit einigen Tagen mit Homematic-Raspi befasst und es nun geschafft raspmatic_rpi2 (3.65.8.20220831) auf einem älteren RPI2B V1.1 den ich noch hatte zu installieren. 
Nun habe ich das HM-MOD-RPI-PCB gekauft, zusammengebaut und auf den PI gesteckt.
Aber wenn ich ein Sensor anlernen will bekommen ich im WebUI die Meldung, dass kein Funkmodul gefunden wird. 
In der Hilfe ist unter HmIP und HmRf alles auf n/a.
Ich habe einige Stunden damit verbracht Informationen zu suchen.
In der Bedienungsanleitung die beilag steht z.B. was von einer Installationsanleitung die bei elv.de zu finden wäre, aber ich ist. 
Im Homatic Forum habe ich auch ein paar Beiträge mit ähnlichen Problemstellungen gefunden, kann aber mit dem was da zu lesen ist nicht sehr viel anfangen.
Es wäre super wenn ich ich ein paar brauchbare Hinweise bekommen könnte.
Herzlichen Dank.

**Comment by Baxxy13** [comment, marked as answer]
> das HM-MOD-RPI-PCB gekauft, zusammengebaut und auf den PI gesteckt.

Der häufigste Fehler ist der, das das eigentliche Funkmodul falsch herum auf die Platine gelötet wurde.
Bilder machen und zeigen, vom Modul allein und aufgesteckt auf den Pi.

**Reply by DiaAlba** [reply]
Da hast du wohl 100%ig Recht. 
Asche auf mein Haupt. Nach dem Hinweis habe ich den BP nochmal verglichen und siehe da .... spiegelverkehrt aufgelötet.
Nach der Nacharbeit wird nun das Modul erkannt und ich konnte die gekauften Sensoren und Aktoren einlernen.
Vielen Dank.
